src/chapters/introduction/about-this-class.rst

```diff
@@ -0,0 +1,67 @@
+=========================
+About LaunchCode Programs
+=========================
+
+Goals
+=====
+
+The goal of our programs is to build our problem solving skills and to learn how to learn.
+Whether you use the coding skills you learn in this program to get a job as a developer is up to you.
+No matter the path you take after this program, learning how to learn will help you continually adapt to the changing needs of your industry.
+
+A goal for us is that we teach you skills to prepare you for jobs in technology.
+We are here to teach you technologies that are prevalent throughout a wide variety of industries.
+
+Blended learning
+================
+
+.. index:: ! blended learning model
+
+We only have a short amount of time in class to learn a lot, so using a **blended learning model** helps us make the most of our time in this class.
+A blended learning model incorporates in-class learning with online materials such as this textbook.
+
+Course activities
+=================
+
+We have spent a lot of time creating the course activities to make the most of your in-class and out-of-class time.
+While there are a lot of different activities we will do in a day, it is important to actively engage with each activity.
+Skipping the prep work or falling behind on assignments can quickly lead to struggling in the class.
+
+**Include a step diagram, noting that skipping any of the components creates a larger gap to cross**
+
+Prep Work
+---------
+
+Prep Work is done before each class session and covers the topics that you will learn about that day. 
+In addition to reading, prep work includes small questions that can help you reinforce your understanding of what you have just read.
+We have found that covering the material for the day before class helps students make the most of their short in-class time.
```

Comment:
Word removal: "We have found that studying the material before class helps students make the most of their short in-class time."

Reply:
Thanks!